lib/feedjira/parser/podcast.rb

```diff
@@ -37,7 +38,7 @@ class Podcast
       element :"itunes:explicit", as: :itunes_explicit
       element :"itunes:complete", as: :itunes_complete
       element :"itunes:new_feed_url", as: :itunes_new_feed_url
-      elements :"itunes:owner", as: :itunes_owners, class: ITunesRSSOwner
+      element :"itunes:owner", as: :itunes_owner, class: ITunesRSSOwner
```

Comment:
maybe leave this as multiple
https://github.com/feedjira/feedjira/blob/master/lib/feedjira/parser/itunes_rss.rb#L60